Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs

```diff
@@ -530,7 +530,7 @@ public void ChangeProgressCounter()
 
         public void UpdateDescription()
         {
-            Description = Id + PullRequestText + " " + DisplayStatus + " " + Duration + TestsResultText;
+            Description = Id + PullRequestText + " " + DisplayStatus + " " + DurationText() + TestsResultText;
```

Comment:
Similar to the comment about `BuildInfo.AddDurationToDescription()` this method doesn't really belong to the DTO. And it should really be a concern of  `AppVeyorAdapter` (in `void UpdateDisplay(...)`).
Since this is an existing method, take this as a general design comment.